integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1574,3 +1574,16 @@ func (s *DockerSuite) TestRunWithNanoCPUs(c *check.C) {
 	c.Assert(err, check.NotNil)
 	c.Assert(out, checker.Contains, "Conflicting options: Nano CPUs and CPU Period cannot both be set")
 }
+
+// #34542
+func (s *DockerSuite) TestRunVolumeLeakage(c *check.C) {
```

Comment:
Pending documentation for this is in #34407

The new suite is `integration/<component>/`, so for this  it would be `integration/container/run_test.go`. Since that doesn't exist yet, you could add it to `integration-cli/docker_api_container)create_test.go`.

Although, looking at this change again. Would it be possible to test this with a unit test?